src/js/stores/MarathonStore.js

```diff
@@ -92,6 +96,22 @@ class MarathonStore extends GetSetBaseStore {
         case ActionTypes.REQUEST_MARATHON_GROUP_CREATE_SUCCESS:
           this.emit(MARATHON_GROUP_CREATE_SUCCESS);
           break;
+        case ActionTypes.REQUEST_MARATHON_GROUP_DELETE_ERROR:
+          let groupErrorMessage = action.data;
+          if (!Object.keys(groupErrorMessage).length) {
```

Comment:
`groupErrorMessage` is going to be emitted as either an Object or a String. Is that handled appropriately in views or should we assemble the string here when we get an Object?